File: app/src/main/java/com/maximus/productivityappfinalproject/domain/GetClosestTimeUseCase.java
package com.maximus.productivityappfinalproject.domain;

import com.maximus.productivityappfinalproject.data.AppsRepository;

public class GetClosestTimeUseCase {
    private AppsRepository mAppsRepository;


    public GetClosestTimeUseCase(AppsRepository appsRepository) {
        mAppsRepository = appsRepository;
    }

    public Long getClosestHour() {
        return mAppsRepository.getClosestHour();
    }

    public Long getClosestDay() {
        return mAppsRepository.getClosestDay();
    }

}

File: app/src/main/java/com/maximus/productivityappfinalproject/domain/GetClosestTimeUseCase.kt
package com.maximus.productivityappfinalproject.domain

import com.maximus.productivityappfinalproject.data.AppsRepository
import javax.inject.Inject

class GetClosestTimeUseCase @Inject constructor(private var appsRepository: AppsRepository){


    fun getClosestHour(): Long {
        return appsRepository.closestHour
    }

    fun getClosestDay(): Long {
        return appsRepository.closestDay
    }
}